src/main/webapp/app/exercises/programming/manage/update/update-components/custom-build-plans/programming-exercise-build-configuration/programming-exercise-build-configuration.component.ts

```
@@ -2,20 +1,80 @@
     styleUrls: ['../../../../programming-exercise-form.scss'],
 })
-export class ProgrammingExerciseBuildConfigurationComponent {
+export class ProgrammingExerciseBuildConfigurationComponent implements OnInit {
+    programmingExercise = input<ProgrammingExercise>();
     dockerImage = input.required<string>();
     dockerImageChange = output<string>();
 
     timeout = input<number>();
     timeoutChange = output<number>();
 
+    envVars: string = '';
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)